client/src/app/project-registration/project-registration-form/project-registration-form.component.ts

```diff
@@ -57,18 +55,28 @@ export class ProjectRegistrationFormComponent implements OnInit, OnDestroy {
               private loaderService: LoaderService,
               private schemaService: SchemaService,
               private autofillProjectService: AutofillProjectService,
-              private cacheProjectService: CacheProjectService
+              private cacheProjectService: ProjectCacheService
   ) {
   }
 
   ngOnInit() {
     const queryParam = this.route.snapshot.queryParamMap;
+    this.setFormData(queryParam);
```

Comment:
I would give this bit of logic a meaningful name.

Reply:
any suggestions on what that could be? this bit of logic does initialize the project form data.

something like `initializeProjectData` or `initializeFormData`?

Reply:
I've changed it to `loadProjectData`

Reply:
I meant the entire content of ngOnInit.